Odata-docs/webapi-8/fundamentals/overview.md

```diff
@@ -0,0 +1,582 @@
+
+# [Visual Studio](#tab/visual-studio)
+
+In the Visual Studio **Package Manager Console**:
+
+```powershell
+Install-Package Microsoft.AspNetCore.OData
+```
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+```dotnetcli
+dotnet add package Microsoft.AspNetCore.OData
+```
+
+---
+
+Internally, ASP.NET Core OData depends on the following.NET libraries for OData: 
+- [`Microsoft.OData.Core`](https://www.nuget.org/packages/Microsoft.OData.Core), which provides features like reading and writing OData payloads, parsing OData URIs and query options
+- [`Microsoft.OData.Edm`](https://www.nuget.org/packages/Microsoft.OData.Edm), which provides support for working with OData schemas, also referred to as EDM (Entity Data Model) model.
+- [`Microsoft.Spatial`](https://www.nuget.org/packages/Microsoft.Spatial), which provides support for geospatial data types.
+
+The following code snippet demonstrates how you would add OData support to your ASP.NET Core application using ASP.NET Core OData in .NET Core 3.1 and .NET 6.0.
+
+# [.NET 6.0](#tab/net60)
+
+```csharp
+// Program.cs
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+// OData namespaces
+using Microsoft.AspNetCore.OData;
+using Microsoft.OData.ModelBuilder;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Create the service EDM model.
+var modelBuilder = new ODataConventionModelBuilder();
+modelBuilder.EntitySet<Customer>("Customers");
+var edmModel = modelBuilder.GetEdmModel();
+
+// Register OData service.
+builder.Services.AddControllers().AddOData(
+    options => options.AddRouteComponents(edmModel));
+
+var app = builder.Build();
+
+app.UseRouting();
+app.UseEndpoints(endpoints => endpoints.MapControllers());
+app.Run();
+```
+
+# [.NET Core 3.1](#tab/netcoreapp31)
+
+```csharp
+// Startup.cs
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+// OData namespaces
+using Microsoft.AspNetCore.OData;
+using Microsoft.OData.ModelBuilder;
+
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        // Create the service EDM model.
+        var modelBuilder = new ODataConventionModelBuilder();
+        modelBuilder.EntitySet<Customer>("Customers");
+        var edmModel = modelBuilder.GetEdmModel();
+
+        // Register OData service.
+        services.AddControllers().AddOData(
+            options => options.AddRouteComponents(edmModel));
+    }
+
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        app.UseRouting();
+        app.UseEndpoints(endpoints => endpoints.MapControllers());
+    }
+}
+```
+
+---
+
+We register OData services using the `AddOData` extension method. The `AddOData` method accepts a callback action that allows us to configure different aspects of the OData service using the options argument (which is an instance of [`ODataOptions`](/dotnet/api/microsoft.aspnetcore.odata.odataoptions)).
```

Comment:
Maybe add a link to the `AddOData` method just like you have done for `ODataOptions` and `AddRouteComponents`